storage/integration_test.go

```diff
@@ -4404,74 +4389,103 @@ func TestIntegration_Scopes(t *testing.T) {
 
 }
 
-func TestIntegration_SignedURL_Bucket(t *testing.T) {
+func TestIntegration_SignedURL_WithCreds(t *testing.T) {
 	ctx := context.Background()
 
-	if testing.Short() && !replaying {
-		t.Skip("Integration tests skipped in short mode")
+	// By default we are authed with a token source, so don't have the context to
```

Comment:
I think this is no longer true, but we should send in explicit credentials still in case the way we authenticate by default changes.